funcs/func.py

```diff
@@ -1,2 +1,12 @@
 def sum(x: float, y: float) -> float:
-    return x + y
\ No newline at end of file
+    return x + y
+
+
+# def misformatted_func(
```

Comment:
I'd remove this but it's good you tested it!

Reply:
Removed.